### PR TITLE
[TACHYON-409] Make tachyon.example a module

### PR DESCRIPTION
**Pull request by haoyuan**
https://tachyon.atlassian.net/browse/TACHYON-409

### CODE REVIEW

**Comment by haoyuan** [comment]
Jenkins, test this please.

**Comment by apc999** [comment]
LGTM, thanks.

- Bin

**Comment by calvinjia** [comment]
+1 LGTM, I think the metrics tests are unstable?

File: examples/pom.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.tachyonproject</groupId>
+    <artifactId>tachyon-parent</artifactId>
+    <version>0.7.0-SNAPSHOT</version>
```
Comment (gpang) [inline examples/pom.xml]: In a future PR, this version should probably be some sort of parameter, to prevent out-of-sync version numbers.

**Pull request merged**